packages.config

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="OSPSuite.Assets" version="9.0.106"/>
-  <package id="OSPSuite.Core" version="9.0.106"/>
-  <package id="OSPSuite.Infrastructure" version="9.0.106"/>
-  <package id="OSPSuite.Infrastructure.Import" version="9.0.106 "/>
-  <package id="OSPSuite.Infrastructure.Autofac" version="9.0.106"/>
-  <package id="OSPSuite.R" version="9.0.106 "/>
+  <package id="OSPSuite.Assets" version="9.0.107"/>

```

Comment:
updating the core also updates the latest version of the potentialParametersForSA. Most specifically, constant parameters with a value of 0 will be ignored